typescript/cli/src/config/warp.ts

```diff
@@ -205,14 +201,27 @@ export function readWarpRouteConfig(filePath: string): WarpCoreConfig {
   return WarpCoreConfigSchema.parse(config);
 }
 
-async function createDefaultWarpIsmConfig(
-  context: CommandContext,
-): Promise<IsmConfig> {
+/**
+ * Creates a default configuration for an ISM with a TRUSTED_RELAYER and FALLBACK_ROUTING.
+ *
+ * Properties relayer and owner are both set as input owner.
+ *
+ * @param owner - The address of the owner of the ISM.
+ * @returns The default Aggregation ISM configuration.
+ */
+async function createDefaultWarpIsmConfig(owner: Address): Promise<IsmConfig> {
```

Comment:
Seems like this function isn't async anymore and need not return a promise, is that right?

Reply:
+

Reply:
yes indeed!

Reply:
Fixed